src/tasks/LibraryBuilder/LibraryBuilder.cs

```diff
@@ -145,6 +155,10 @@ public override bool Execute()
 
         if (UsesRuntimeInitCallback && !UsesCustomRuntimeInitCallback)
         {
+            if (BundlesResources && BundledRuntimeConfig.Length == 0)
+            {
+                throw new LogAsErrorException($"'{nameof(BundledRuntimeConfig)}' is required when bundling and using the default runtime callback.");
```

Comment:
Do we really need to be this strict? When not using bundling we have logic in our auto init logic to just skip call to monovm_runtimeconfig_initialize, I believe we should do the same if you don't bundle a runtime config. In that case you can set the %RUNTIME_CONFIG_DATA% and %RUNTIME_CONFIG_DATA_LEN% to NULL and 0 and ignore call to monovm_runtimeconfig_initialize. Behavior should be the same when running library builder with or without bundling support.